apps/wakunode2/wakunode2_setup_rest.nim

```diff
@@ -13,7 +13,7 @@ import
 
 
 logScope:
-  topics = "wakunode.setup.rest"
+  topics = "wakunode rest"
```

Comment:
I know not part of this PR, but FWIW I like that the app is just `wakunode` like here - I'm getting a bit wary of having a confusing `2` everywhere, when we're practically always working with version 2.

Reply:
Yes... I also have that concern. To achieve that, we need to coordinate with the infra changes. I have avoided mixing too many infra changes for this release, we are already changing the configuration.

The related issue is: https://github.com/status-im/nwaku/issues/1002